e2e/testcases/resource_conditions_test.go

```diff
@@ -140,14 +145,18 @@ func TestResourceConditionAnnotations(t *testing.T) {
 		`configmanagement.gke.io/errors-`)
 
 	// Ensure error conditions are removed.
-	_, err1 = nomostest.Retry(20*time.Second, func() error {
-		return nt.Validate(repo.DefaultName, "", &v1.Repo{},
+	_, err1 = nomostest.Retry(waitTimeout, func() error {
+		return nt.Validate("repo", "", &v1.Repo{},
```

Comment:
Do we expect delay between these status updates? Or would it be sufficient to wrap them all in the same retry?

Reply:
Oops, it was merged before I saw the comment. I think wrap them all in the same retry is a good suggestion. We can make a separate PR.